runelite-client/src/main/java/net/runelite/client/plugins/config/ConfigPanel.java

```diff
@@ -852,35 +852,35 @@ else if (cid.getType() == String.class)
 					textField.setBorder(BorderFactory.createEmptyBorder(5, 5, 5, 5));
 					textField.setText(configManager.getConfiguration(cd.getGroup().value(), cid.getItem().keyName()));
 
-					textField.addFocusListener(new FocusAdapter()
+					DeferredDocumentChangedListener listener = new DeferredDocumentChangedListener();
+					listener.addChangeListener(e ->
 					{
-						@Override
-						public void focusLost(FocusEvent e)
+						ConfigItem configItem = cid.getItem();
+						if (configItem.parse())
 						{
-							ConfigItem item = cid.getItem();
-							if (item.parse())
-							{
-								Boolean result = parse(item, textField.getText());
+							Boolean result = parse(configItem, textField.getText());
 
-								if (result != null && result)
-								{
-									changeConfiguration(listItem, config, textField, cd, cid);
-								}
-							}
-							else
+							if (result != null && result)
 							{
+								log.info("Saving config after debounce called!");
```

Comment:
debug rather than info as well pls

Reply:
Completely forgot to remove the debugging